components/icons/images/tier2/edit-not-editable.svg

```diff
@@ -0,0 +1,17 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+  <defs>
+    <clipPath id="a">
+      <path d="M0 0h24v24H0z"/>
+    </clipPath>
+  </defs>
+  <g clip-path="url(#a)" data-name="24x24-no-editing-icon">
+    <path fill="#fff" d="M0 0h24v24H0z"/>
+    <path fill="#494c4e" stroke="#565a5c" stroke-linecap="round" stroke-width="2" d="M2.5 4.5l17 17" data-name="Path 3554" opacity=".998"/>
```

Comment:
Should `stroke="#565a5c"` actually be #494c4e? I'm worried that when a different colour is applied to this icon that grey bit will be missed.

Reply:
Oops must've been something the designer missed, I can fix that

Reply:
That looks like the old Ferrite color.  Might be something they want to remove from where they are picking that from.

Reply:
I can let him know!